ui-ngx/src/app/shared/components/js-func-modules.component.html

```diff
@@ -27,7 +27,7 @@
         </div>
         <div *ngIf="modulesFormArray().controls.length; else noModules" class="tb-form-table-body">
           <div *ngFor="let moduleControl of modulesFormArray().controls; trackBy: trackByModule; let $index = index;">
-            <tb-js-func-module-row class="flex-1"
+            <tb-js-func-module-row class="flex flex-1"
```

Comment:
Class flex-1 unnecessary. 
The display: block property on the parent element prevents flex from having an effect.